code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -551,6 +555,7 @@
 		/datum/gas/tritium = 30,
 		/datum/gas/bz = 20,
 		/datum/gas/nitryl = 30,
+		/datum/gas/plasma = MINIMUM_MOLE_COUNT,
```

Comment:
why you added plasma as requirement?

Reply:
It has always been a requirement, just not explicitly. The plasma # is hooked to a min() proc for the moles formed. I thought that formally putting it there might suit the new react system better, thought i didn't give much thought into it.